_includes/footer.html

```diff
@@ -18,7 +18,10 @@
     </a>
   </div>
   <div id="copyright" class="fx-col-100-xs fx-row-center-xs">
-    {{ site.data.general.footer.copyright }}
+	Software by <a href="https://github.com/corrados" target="_blank" rel="noreferrer">Volker Fischer</a> and <a href="https://github.com/jamulussoftware/jamulus/graphs/contributors" target="_blank" rel="noreferrer">contributors</a> - want to <a href="/wiki/Contribution">get involved?</a> 
+	<div id="asio" style="margin-top:20px;">
```

Comment:
Should this really be nested inside the `copyright` div or a separate `asio` div (as it's not an assertion but an acknowledgement - two distinct entities)?

Reply:
Dunno. But I was expecting suggestions, certainly.